JDK language specific tests

When checking out the current master branch and do a build,
some tests fail because the expected error message is in english but my system is not in english so the actual error message does not match.

Example:
`[ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 5.375 s <<< FAILURE! - in io.quarkus.reactive.pg.client.ReactivePgReloadTest
[ERROR] testHotReplacement  Time elapsed: 5.365 s  <<< FAILURE!
java.lang.AssertionError: 
1 expectation failed.
Response body doesn't match expectation.
Expected: (a string starting with "Connection refused" and a string ending with ":2345")
  Actual: Verbindungsaufbau abgelehnt: localhost/127.0.0.1:2345
`

Running maven with -Duser.language=en -Duser.country=US did not solve the problem.

--- a/extensions/reactive-pg-client/deployment/src/test/java/io/quarkus/reactive/pg/client/LegacyReactivePgReloadTest.java
+++ b/extensions/reactive-pg-client/deployment/src/test/java/io/quarkus/reactive/pg/client/LegacyReactivePgReloadTest.java
@@ -25,7 +25,7 @@ public class LegacyReactivePgReloadTest {
                 .get("/dev/error")
                 .then()
                 .statusCode(200)
-                .body(Matchers.allOf(Matchers.startsWith("Connection refused"), Matchers.endsWith(":2345")));
+                .body(Matchers.endsWith(":2345"));
 
         test.modifyResourceFile("application.properties", s -> s.replace(":2345", ":5234"));
 
@@ -33,6 +33,6 @@ public class LegacyReactivePgReloadTest {
                 .get("/dev/error")
                 .then()
                 .statusCode(200)
-                .body(Matchers.allOf(Matchers.startsWith("Connection refused"), Matchers.endsWith(":5234")));
+                .body(Matchers.endsWith(":5234"));
     }
 }

